ParkingZoneApp/Program.cs

```diff
@@ -56,11 +58,15 @@ public static void Main(string[] args)
             app.MapControllerRoute(
                 name: "Admin",
                 pattern: "{area:exists}/{controller=ParkingZone}/{action=Index}/{id?}");
-
+            
```

Comment:
space

Reply:
![image](https://github.com/SamanKingDotNetDeveloper/Parking_Zone_Client/assets/100861214/0e10c120-04d5-44a6-8072-1720d67d9516)
In visual studio it is not showing any spaces

Reply:
just delete this row and add again with enter